### DISCUSSION TITLE
CKEditor: Adding source editing button

### DISCUSSION THREAD

**Discussion by SugarShaker** [Laravel-Backpack/community-forum#685] [Q&A (Help)] [answered]
How do I add the source edit button on the ckeditor navbar.
In ckeditor documentation its:
```javascript
import { SourceEditing } from '@ckeditor/ckeditor5-source-editing';

ClassicEditor
    .create( document.querySelector( '#editor' ), {
        plugins: [ SourceEditing, /* ... */ ],
        toolbar: [ 'sourceEditing', /* ... */ ]
    } )
    .then( /* ... */ )
    .catch( /* ... */ );
``` 
Is there a way to do this in backpack?
I also need to allow div and class tags, once I can edit the source. 

Thanks,

Rob

**Comment by karandatwani92** [comment, marked as answer]
To add more plugins to CKEditor follow the guide [here](https://github.com/Laravel-Backpack/community-forum/issues/559#issuecomment-1630673757)

**Reply by SugarShaker** [reply]
Great... many thanks.
Do you know of any guides on how to allow different tags and attributes on tags, so ckeditor doesn't strip them out?
